### PR TITLE
feat: Extend the model of `Segment` with additional properties (DEV-3505)

### PR DESCRIPTION
**Pull request by BalduinLandolt**
### Task Description/Number

Issue Number: DEV-3505

This PR adds a number of properties to `knora-base:Segment` in hopes that most projects will find their needs satisfied by this, rather than needing subclasses of Segment with custom properties.  
For all of them, mappings to popular external ontologies have been added, in hope that this will be a step towards making data more interoperable; furthermore this backs up the claim that this is supposed to be a reasonable set of properties for the concept of segments.

### PR Type

- [ ] build/chore: maintenance tasks (no production code change)
- [ ] docs: documentation changes (no production code change)
- [ ] feat: represents new features
- [ ] fix: represents bug fixes
- [ ] perf: performance improvements
- [ ] refactor: represents production code refactoring
- [ ] test: adding or refactoring tests (no production code change)
- [ ] deprecated: Deprecation warning (ideally referencing a migration guide)

### Basic requirements for bug fixes and features

- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

### Does this PR introduce a breaking change?

- [ ] Yes

### Does this PR change client-test-data?

- [ ] Yes

### CODE REVIEW

File: webapi/src/main/resources/knora-ontologies/knora-base.ttl [hunk 1]
```diff
@@ -2327,7 +2405,22 @@
...
                        [ rdf:type           owl:Restriction ;
                          owl:onProperty     :hasSegmentBounds ;
-                         owl:cardinality    "1"^^xsd:nonNegativeInteger ] ;
+                         owl:cardinality    "1"^^xsd:nonNegativeInteger ] ,
+                       [ rdf:type           owl:Restriction ;
+                         owl:onProperty     :hasTitle ;
+                         owl:maxCardinality "1"^^xsd:nonNegativeInteger ] ,
+                       [ rdf:type           owl:Restriction ;
+                         owl:onProperty     :hasDescription ;
+                         owl:minCardinality "0"^^xsd:nonNegativeInteger ] ,
```
Comment (seakayone) [inline webapi/src/main/resources/knora-ontologies/knora-base.ttl:2414]: **question**: A `Segment` may have more than one description?

I would have expected a single description to be optional.

**Review by seakayone — APPROVED** [review]

File: webapi/src/main/resources/knora-ontologies/knora-base.ttl (hunk 1 above)
Comment (BalduinLandolt) [reply webapi/src/main/resources/knora-ontologies/knora-base.ttl:2414]: That was my first assumption too, but turns out that projects may want to be able to describe different aspects of a Segment as different descriptions. I'd have been happy either way, but this aligns with how EKWS modelled their Sequences, which I found reason enough to go that way.

**Review by mpro7 — APPROVED** [review]
Should this be merged w/o added documentation, knowing that Friday the release will happen?

**Comment by BalduinLandolt** [comment]
> Should this be merged w/o added documentation, knowing that Friday the release will happen?

looks like this is approved, so I can write documentation for it right away

**Review by siers — APPROVED** [review]
schema.org – interesting.

**Pull request merged**